### DISCUSSION TITLE
请问，其他主机的docker，如何实现控制和监测资源占用

### DISCUSSION THREAD

**Discussion by sidneyelliottl8** [hslr-s/sun-panel#290] [Q&A]
如题。

**Comment by wintsa123** [comment]
docker.sock的那个文件，用smb之类的，共享给sunpanel应该可以，但目前的sunpanel设定是只监听一个docker.sock。

**Comment by Chenghao999** [comment]
把你想要监控的docker文件里的 docker.sock 文件找到，目前只支持监控一个
